File: Perez-Meneses-Juan-Carlos/Procesos/execl.c
#include <stdio.h>
#include <unistd.h>
#include <sys/types.h>

int main()
{
	pid_t pid = getpid();
	printf("El pid actual es: %d\n\n",pid);
	execl("/usr/bin/ps", "ps","-fea" , (char *)NULL);
	{
		printf("Nunca me veran");
	}
	return 0;
}

File: Perez-Meneses-Juan-Carlos/Procesos/execl.py
import os
import subprocess

# python ./execl

pid = os.getpid()
print(f"El pid actual es: {pid}\n")

try:
    os.execl("/bin/ps", "ps", "-fea")
except Exception as e:
    print(f"Error al ejecutar execl: {e}")
